src/configs/mod.rs

```diff
@@ -164,6 +165,7 @@ pub enum PeerFlag {
     NixOpsMachine,
     Center,
     Template,
+    UseTemplate { peer: u128 },
```

Comment:
Having It bind to name instead would make for a better json editing experience.

Reply:
Done